dvc/command/ls/ls_colors.py

```diff
@@ -0,0 +1,61 @@
+import os
+
+from dvc.compat import fspath
+
+
+class LsColors(object):
+    default = "rs=0:di=01;34:ex=01;32"
+
+    def __init__(self, lscolors=None):
+        self._extensions = {}
+        self._codes = {}
+        if lscolors:
+            self.load(lscolors)
+
+    def clear(self):
+        self._codes.clear()
+        self._extensions.clear()
+
+    def load(self, lscolors):
+        self.clear()
+
+        if not lscolors:
+            return
+
+        for item in lscolors.split(":"):
+            try:
+                code, color = item.split("=", 1)
+            except ValueError:
+                continue
+            if code.startswith("*."):
+                self._extensions[code[1:]] = color
+            else:
+                self._codes[code] = color
+
+    def load_from_environ(self, envvar="LS_COLORS", default=None):
+        return self.load(os.environ.get(envvar, default))
+
+    def format(self, fs_node, text=None):
+        text = text or fspath(fs_node["path_info"])
+
+        if fs_node.get("isdir", False):
+            return self._format(text, code="di")
+
+        if fs_node.get("isexec", False):
+            return self._format(text, code="ex")
+
+        _, ext = os.path.splitext(text)
+        if ext:
+            return self._format(text, ext=ext)
+        return text
```

Comment:
Do we color outs?

Reply:
We don't color them for now to avoid overlapping with themes. We'll find a better way to mark outs later.

Reply:
I think @Suor mentions a general opportunity, @efiop 
I removed completely the logic with matching `dvc` outputs, what I need to recover I think (without forcing color)